server/datastore/mysql/scripts.go

```diff
@@ -1039,7 +1039,14 @@ func buildHostLockWipeStatusUpdateStmt(refCol string, succeeded bool, joinPart s
 			// Note that this must not clear the unlock_pin, because recording the
 			// lock request does generate the PIN and store it there to be used by an
 			// eventual unlock.
-			stmt += fmt.Sprintf("%sunlock_ref = NULL, %[1]swipe_ref = NULL", alias)
+			if !setUnlockRef {
+				// Currently only used for Apple MDM devices.
+				// We set the unlock_ref to current time since the device can be unlocked any time after the lock.
+				// Apple MDM does not have a concept of unlock pending.
```

Comment:
Just a thought: Am guessing we won't need this for iOS/iPadOS, but if so, we can add a new column (as `unlock_ref` will now contain time instead of a command UUID).

Reply:
`unlock_ref` only contains time for Apple devices. It contains UUID for Windows. This is how it was coded originally by MDM team.